subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -178,9 +178,6 @@ This can also be followed by a `because` text.
 
 | Rejection: version <version>: <attributes information>
 | The dependency has a dynamic version, and some versions did not match the <<variant_model#sec:variant-aware-matching, requested attributes>>.
-
-| Forced
-| The build <<dependency_downgrade_and_exclude#forced_dependencies_vs_strict_dependencies, enforces>> the version of the dependency.
```

Comment:
Don't we still need this text? I think this is related to [Louis' comment?](https://github.com/gradle/gradle/pull/21916/files#r965626871)

Reply:
You're right, this is still printed when forced by `resolutionStrategy.force`.